### PR TITLE
[UI] change review status icons on pr view style to github style

### PR DESCRIPTION
**Pull request by a1012112796**
I think the github style is more meaningfull, so make this chang
* change the icon of ApproveReview pr from "eye" to "check" like github
* change the icon of RejectReview pr from "x" to "request-change" like github

view:    

jta

### CODE REVIEW

File: templates/repo/issue/view_content/comments.tmpl [hunk 1]
```diff
@@ -318,7 +318,11 @@
 		</div>
 	{{else if eq .Type 22}}
 		<div class="event" id="{{.HashTag}}">
-			<span class="issue-symbol">{{svg (printf "octicon-%s" .Review.Type.Icon) 16}}</span>
+			{{svg "octicon-primitive-dot" 16}}
+			<span class="issue-symbol {{if eq .Review.Type 1}}green
+			{{else if eq .Review.Type 2}}grey
+			{{else if eq .Review.Type 3}}red
+			{{else}}grey{{end}}">{{svg (printf "octicon-%s" .Review.Type.Icon) 16}}</span>
```
Comment (silverwind) [inline templates/repo/issue/view_content/comments.tmpl]: Would just put this on a single line to avoid newlines in the output.

**Comment by silverwind** [comment]
Got some before/after screenshots?

File: templates/repo/issue/view_content/comments.tmpl (hunk 1 above)
Comment (guillep2k) [reply templates/repo/issue/view_content/comments.tmpl]: Alternatively, go templates allow [the following syntax](https://golang.org/pkg/text/template/#hdr-Text_and_spaces):

```
			<span class="issue-symbol {{if eq .Review.Type 1}}green
			{{- else if eq .Review.Type 2}}grey
			{{- else if eq .Review.Type 3}}red
			{{- else}}grey{{end}}">{{svg (printf "octicon-%s" .Review.Type.Icon) 16}}</span>
```

(hint: check the `- ` preceeding the `else`)

**Comment by lunny** [comment]
Screenshots are welcome.

**Comment by a1012112796** [comment]
Sorry , I forgot it.
This change have two main part
* first , change icon of approve from ``eye`` to ``check`` and change the icon for Reject from "x" to ``request-change``. because I think the github style iss more meaningfull.
the compare view is bellow
now used:
jtd
changed to:
jtf

after this change , I found that these icons include "eye" , "check" , "lock" and "key" can't bound at the timeline like the  "primitive-dot", but the github can, but I don't know how to do it, so just add a additional "primitive-dot" to make these item onto the time-line 
now view:
jtb
may be it will have more good way.
The github:
https://github.com/go-gitea/test_repo/pull/4

File: templates/repo/issue/view_content/comments.tmpl (hunk 1 above)
Comment (a1012112796) [reply templates/repo/issue/view_content/comments.tmpl]: sorry , I just copy it from other place. you are right , I will change it

**Comment by a1012112796** [comment]
Hello, Are these changes usefull?Looking forward to replys , Thanks .

**Comment by silverwind** [comment]
New icons seem reasonable. I wonder if there is a better solution for this though:

image

Is the example just a neutral review or a negative one? If neutral, the icon should be grey ideally.

**Comment by lunny** [comment]
I like the new icons. Thanks! @a1012112796

**Comment by a1012112796** [comment]
@silverwind  hello, thanks for your suggestions, I will do them soon, but not only the ``request-change``
can't on time line, but also ``check`` , ``lock``  and ``key`` also can't into the time-line.
If it can't be solve, I have another suggest style for them , that move them to the end of the line,
and add a single dot at front of all type items to bound the time line, how about this way?
jth

**Comment by a1012112796** [comment]
@silverwind Hello , I have try your suggestion. But it is seem has some problems , that can't change the color of ``primitive-dot`` , because It has defined as green in somewhere , but I don't know how to change it , I haven't study css before :cry: . can you help me about  how to  solve it, Thanks.

jtd

**Comment by a1012112796** [comment]
now, I just change the icons style . for the problem about some items can't bound on the time-line, left it and wait other contributors who can solve it 😄

**Comment by a1012112796** [comment]
Hello, How about these changes now? Looking forward to replys , Thanks .

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
> for the problem about some items can't bound on the time-line

I can't seem to figure it out either. Can't see any style that would cause it, it's weird.

**Comment by guillep2k** [comment]
@a1012112796 could you please update the screenshots?

**Comment by a1012112796** [comment]
@guillep2k hello, these screenshots are same as now, all not right screenshots which are add by me has been deleted.
https://github.com/go-gitea/gitea/pull/10737#issuecomment-599965937
https://github.com/go-gitea/gitea/pull/10737#issue-389233249

**Review by guillep2k — APPROVED** [review]

**Comment by a1012112796** [comment]
I wander the status of PR now ...

**Pull request merged**